.github/workflows/deploy-canisters.yml

```diff
@@ -42,13 +37,17 @@ jobs:
 
                   if [[ "$TARGET_BRANCH" == "main" ]]; then
                     echo "environment=production" >> $GITHUB_OUTPUT
```

Comment:
This is sometimes called `network` and sometimes `environment`. Use a single term everywere